Assets/Plugins/EuclideonUdSDK/udSDK/csAPI/udSDKAPI.cs

```diff
@@ -509,17 +535,42 @@ public RenderOptions() : this(udRenderContextPointMode.udRCPM_Rectangles, udRend
 
         public void setPick(uint x, uint y)
         {
-            pick = new udRenderPicking();
+            udRenderPicking pick = new udRenderPicking();
             pick.x = x;
             pick.y = y;
-            Marshal.StructureToPtr(pick, options.pPick, true);
+
+            Marshal.StructureToPtr(pick, options.pPick, false);
 
             pickRendered = false;
             pickSet = true;
+        }
 
+        public udPick getPick()
+        {
+            UnityEngine.Debug.Log("Getting pick");
+
+            if(!pickSet)
+                return new udPick();
+
+            udRenderPicking targetPick = this.Pick;
+            udPick newPick = new udPick();
+            newPick.hit = (int)targetPick.hit ;
```

Comment:
Remove space before `;`. There are several places in this PR with a space before the statement terminator `;`